mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/TransferFragment.java

```diff
@@ -111,6 +113,26 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         mPresenter.fetchMobile();
 
         mEtMobileNumber.addTextChangedListener(new PhoneNumberFormattingTextWatcher());
+        etAmount.addTextChangedListener(new TextWatcher() {
+            @Override
+            public void beforeTextChanged(CharSequence s, int start, int count, int after) {
+
+            }
+
+            @Override
+            public void onTextChanged(CharSequence s, int start, int before, int count) {
+                if (s.toString().contains(".") &&
```

Comment:
Try to add some documentation here. It would be difficult for someone who isn't aware of this PR what this code is doing. You can also make `.` a constant like `AMOUNT_DECIMAL_SEPARATOR`